tests/storage/memory_dump/conftest.py

```
@@ -5,28 +5,35 @@
-    golden_image_data_source_scope_function,
+    modern_cpu_for_migration,
 ):
-    with vm_instance_from_template(
-        request=request,
-        unprivileged_client=unprivileged_client,
-        namespace=namespace,
-        data_source=golden_image_data_source_scope_function,
-    ) as vm:
-        running_vm(vm=vm)
+    with (
+        create_windows2022_dv_template_from_registry(
```

Ruff (0.15.20)

[error] Undefined name `create_windows2022_dv_template_from_registry`

(F821)